src/components/_classes/component/Component.js

```diff
@@ -2391,6 +2392,14 @@ export default class Component extends Element {
       return false;
     }
 
+    if (flags.changed && !flags.changed.value) {
+      return false;
+    }
+
+    if (flags.changed && row.label && flags.changed.value !== row.label) {
```

Comment:
And what if they have a field in their form named "Label"?  This will cause a "magic" bug that will only break if you name a textfield Label. Seems a little precarious.